pkg/util/service/service.go

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
Thoughts on moving this to https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/cloud-provider/service?

Reply:
We could but this is a general util file so putting it there could eventually mean duplication of code. The reason I put it here is there are similar utilities in the util package for other resources.